book/scripts.md

```diff
@@ -1067,6 +1067,32 @@ def add_entry(params):
     return show_comments()
 ```
 
+::: {.further}
+
+Search engines are constantly [crawling] the web and [indexing] all of the web
+pages they can find. In the early days, indexing was just a matter of loading
+the HTML, parsing it and extracting the information. But these days, a lot of
+[single-page app][spa] sites use JavaScript to ["hydrate"][hydration]
+[^why-hydrate] their site into its full contents. On such sites, before
```

Comment:
I think the placement of `[^why-hydrate]` here adds an extra space between that and `"hydrate"`, which is not right.